### ISSUE TITLE
[Bug]: (Crash immediately after 'Scan recent videos for a channel' option.)

### ISSUE THREAD

**Issue by sanikdah** [ThioJoe/YT-Spammer-Purge#224]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

Well, I was trying to use the "Scan recent videos for a channel" option on the main menu but then I get an error message.

What did you expect to happen?

Well, I expected it to pop up with some sort of menu and allow me to scan videos from a channel.

### Release version

v2.2.6

### Steps to reproduce

1. Open the app.
2. Set everything up.
3. Use the attached config file (obviously renamed to .ini [GitHub doesn't let me attach an ini file] )
4. Press '2' on the main menu screen.
5. It crashes.
[SpamPurgeConfig.txt](https://github.com/ThioJoe/YouTube-Spammer-Purge/files/7778718/SpamPurgeConfig.txt)

### What platform are you seeing this problem on?

Windows

### Relevant log output

```shell
Error that was shown in the console:
Choice (1-6): 2
Traceback (most recent call last):
  File "C:\Users\Joe\Documents\GitHub\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 2776, in <module>
  File "C:\Users\Joe\Documents\GitHub\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 2200, in main
KeyError: 'channel_to_scan'
[16928] Failed to execute script 'YouTubeSpammerPurge' due to unhandled exception!

[process exited with code 1]
```

**Comment by KendallDoesCoding** [comment]
@sanikdah It meant to cause a error because the line 2776 makes it cause a error:
try:
    main()
  except HttpError as e:
    traceback.print_exc()
    print("------------------------------------------------")
    print("Error Message: " + str(e))

This can be caused, by not proper setup. Idk but you may have to run the program first before you can straight away run the config file.

Also, ensure you got your client_secrets.json file.

**Comment by rbeer** [comment]
I guess you have a manually edited config file with `channel_to_scan` removed.

You need this entry in your YouTubeSpamPurge.ini:
https://github.com/ThioJoe/YouTube-Spammer-Purge/blob/85624197091eb0cc9c23b1114024fc90d52f159e/assets/default_config.ini#L65

**Comment by ThioJoe** [comment]
Yea you're using an old version of the config file, delete it and have the program regenerate one. In the latest versions, it will warn you if you your config file is out of date from now on.

**Comment by ThioJoe** [comment]
oops turns out this is my fault, the latest exe release (2.2.6) was packaged with an old version of the default config by mistake

**Comment by ThioJoe** [comment]
Ok should be fixed in latest release, 2.3.0
